plugins/ruby-foreman-tasks/debian/gem.list

```diff
@@ -1,3 +1,3 @@
 # list of gem urls to download via Jenkins, space separated
-GEMS="https://rubygems.org/downloads/foreman-tasks-7.2.1.gem"
+GEMS="https://rubygems.org/downloads/foreman-tasks-8.0.1.gem"
 GEMS="$GEMS https://rubygems.org/downloads/parse-cron-0.1.4.gem"
```

Comment:
This will require updates for fugit, but our Debian bumping doesn't do it automatically yet